dockerfiles/Dockerfile-archlinux

```diff
@@ -0,0 +1,162 @@
+    --disable-dependency-tracking \
+    --disable-upower-glib \
+    --disable-static \
+    --enable-maintainer-mode --enable-systemd --enable-gtk-doc \
+    --enable-xrandr --enable-xcursor --enable-libnotify \
+    --enable-epoxy --enable-startup-notification --enable-xsync \
+    --enable-render --enable-randr --enable-xpresent \
+    --enable-compositor --enable-libxklavier --enable-pluggable-dialogs \
+    --enable-sound-settings --enable-polkit --disable-network-manager \
+    --enable-notifications --enable-gio-unix --enable-gudev \
+    --enable-exif --enable-pcre --enable-vala \
+    --enable-introspection --enable-sound-settings --enable-gio-unix \
+    --with-vala-api=0.36 \
+    --with-perl-options=INSTALLDIRS=vendor}"
+
+# allow extraction of man pages?
+RUN if (( ${EXTRACT_MAN/#true/1} )); then \
+        temp_conf="$(cat /etc/pacman.conf)"; \
+        /bin/bash -c 'while IFS= read -r line; do \
+            if [[ ! $line =~ ^NoExtract.*usr/share/man.*$ ]]; then \
+                echo "$line"; \
+            fi; \
+        done;' <<< $temp_conf >/etc/pacman.conf; \
+    fi
+
+RUN pacman -Syu --noconfirm \
+    && pacman -S base-devel git ${USERSHELL} man-db --noconfirm --needed
+
+# Setup the test user
+RUN useradd --create-home --no-log-init --shell "/bin/${USERSHELL}" "${DEFAULT_USER}"
+# RUN usermod --add-subuids 10000-65536 --add-subgids 10000-65536 "${DEFAULT_USER}"
+# RUN echo "${DEFAULT_USER}:10000:65536" >> /etc/subuid
+# RUN echo "${DEFAULT_USER}:10000:65536" >> /etc/subgid
+RUN install -dm755 /etc/sudoers.d/
+RUN { \
+    echo "Defaults:%${DEFAULT_USER} targetpw"; \
+    echo "%${DEFAULT_USER} ALL=(ALL) NOPASSWD: ALL"; \
+} > /etc/sudoers.d/20-xfce-test-user
+
+ENV USER_HOME="/home/${DEFAULT_USER}"
+
+# for makepkg
+ENV PACKAGER="${DEFAULT_USER} <xfce4-dev@xfce.org>"
+ENV BUILDDIR=/var/cache/makepkg-build/
+RUN install -dm755 --owner=${DEFAULT_USER} ${BUILDDIR}
+
+RUN runuser -u "${DEFAULT_USER}" -- git clone https://aur.archlinux.org/${PACMAN}-bin.git /tmp/${PACMAN}
+RUN cd /tmp/${PACMAN} \
+    && sudo runuser -u "${DEFAULT_USER}" -- makepkg --install --force --syncdeps --rmdeps --noconfirm --needed
+
+# install more packages required for the next few steps
+RUN runuser -u ${DEFAULT_USER} -- ${PACMAN} -S python-behave gsettings-desktop-schemas --noconfirm --needed
+
+# needed for LDTP and friends
+RUN /usr/bin/dbus-run-session /usr/bin/gsettings set org.gnome.desktop.interface toolkit-accessibility true
+
+# copy in our scripts
+COPY --chown=$DEFAULT_USER container_scripts /container_scripts
+RUN chmod +x /container_scripts/*.sh /container_scripts/*.py
+
+# xfce specific build dependencies and default panel plugins
+RUN /container_scripts/build_time/install_packages-${TAG}.sh
+
+# Install _all_ languages for testing
+# RUN ${PACMAN} -Syu --noconfirm \
+#  && ${PACMAN} -S transifex-client xautomation intltool \
+#     opencv python-google-api-python-client \
+#     python-oauth2client --noconfirm --needed
```

Comment:
This will be needed to allow testing to work. It should be added later when the image builds